zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -108,8 +108,8 @@ public Notebook(
     this.noteSearchService = noteSearchService;
     this.notebookAuthorization = notebookAuthorization;
     this.credentials = credentials;
-    quertzSchedFact = new org.quartz.impl.StdSchedulerFactory();
-    quartzSched = quertzSchedFact.getScheduler();
+    StdSchedulerFactory quartzSchedFact = new StdSchedulerFactory();
+    quartzSched = quartzSchedFact.getScheduler();
```

Comment:
```suggestion
    quartzSched = new StdSchedulerFactory().getScheduler();
```
Проще, короче, но не существенно. Так в рамках красоты =)